libcontainer/nsenter/nsexec.c

```diff
@@ -150,76 +146,60 @@ static uint8_t readint8(char *buf)
 
 static void update_process_idmap(char *pathfmt, int pid, char *map, int map_len)
 {
-	char    buf[PATH_MAX];
-	int	len;
-	int	fd;
-
-	len = snprintf(buf, sizeof(buf), pathfmt, pid);
-	if (len < 0) {
-		pr_perror("failed to construct '%s' for %d", pathfmt, pid);
-		exit(1);
-	}
+	int	fd, len;
+	char path[PATH_MAX];
 
-	fd = open(buf, O_RDWR);
-	if (fd == -1) {
-		pr_perror("failed to open %s", buf);
-		exit(1);
-	}
+	len = snprintf(path, PATH_MAX, pathfmt, pid);
+	if (len < 0)
+		bail("failed to construct '%s' for %d", pathfmt, pid);
+
+	fd = open(path, O_RDWR);
+	if (fd < 0)
```

Comment:
What do you mean? I'm just matching the kernel style (where single-line statements don't need wrapping braces).